src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -255,6 +257,17 @@ correspond to the `FlipAngle` of `35`, `10` and `25` degrees).
 | echo    | 1,2,... N  | EchoTime            |
 | tsl     | 1,2,... N  | SpinLockTime        |
 
+For example (for a multi-echo FLASH dataset):
+
+```Text
```

Comment:
Out of curiosity, is this example intended for T2* mapping? If this is the case would you rather prefer using a suffix like `MET2star` (or something more convenient), or do you think that sticking with `T1w` is better? My only concern is that contrast differences between earlier/later echoes may not be ideal for interchangeability of these images attaining `T1w` suffix.

Reply:
This example is based on "BEP001 example dataset 01 (multi-echo FLASH)", which uses FLASH suffix. However, FLASH is not an agreed suffix AFAIK. MESET2 would be a close one; however, FLASH is a GRE rather than SE. I think these images can still be considered "predominantly T1-weighted" (see [here](http://mriquestions.com/multi-echo-gre.html)), that's why I decided for T1w.

Reply:
In previous discussions staying clear of sequence names (especially for sequences like FLASH/MPRAGE) as suffixes was a prominent remark given that vendor-specific names are not desirable for inclusiveness and high versatility of such sequences can be a confounder for apps attempting to pick up a set of images. 

As for weighting of echo images, especially for TE ranges that span a wide interval e.g. from 2ms to 30ms+, longer echoes will be more T2*w and the contrast differences will be perceivable.

Letting alone the potential differences based on the flip angle choice, I would side with using an alternative (grouping) suffix for good measure. This also better describes the intention why these images are present in the dataset.

Reply:
Which (existing) suffix do you suggest, then?

Reply:
Really useful pointer, thank you! I fail to see if any of the existing suffixes can be useful here. I suggest we define a new one. 

Conceptually, the closest one we have right now is `MESET2`. So, maybe we can create a new suffix `MEGRE` and explain possible use cases in the table (e.g. multi-echo FLASH {SPGR, FFE} can be used for T2star mapping or FreeSurfer's MEF analysis).

Reply:
I like `MEGRE`. Shall we seek some consensus on it, or shall I simply update the PR?

Reply:
Pinging @KirstieJane @Gilles86 :)

Reply:
@emdupre is this what we were talking about today? Do you use a similar dataset for calculating weighted average image from multi-echo images?

Reply:
MEGRE is great alternative for FLASH afaic.

Reply:
How about adding the MEGRE suffix with this definition:

Groups together parametrically linked multiple anatomical gradient echo images acquired at different echo times. The `<indexable_metadata>-<index>` key/value pair of `echo-<index>` is REQUIRED for the images grouped by this suffix. Associated output suffixes can be: T2starmap, R2starmap, when used for T2* mapping.

Reply:
@tiborauer +1. We can explain more use cases (e.g. weighted averaging @emdupre) and make it even more inclusive.

Reply:
Added

Reply:
This is a really useful discussion, thank you ! I do think I'm a bit confused, though:

In tedana we take in the initial time series for each echo and calculating a T2* map, then perform a voxelwise weighted averaging based on the T2* values. We then use that weighted average (we call it "optimally combined") time series to do several denoising steps.

So, in that context, would you recommend that we recommend our intake files have an `_MEGRE` suffix ? Right now, we look for `_bold` suffix files. In fact, it was originally this point (that multi-echo files _are_ bold files, just with "unusual" echo times) that Chris and I converged on when getting multi-echo functional data into the main BIDS specification.

Reply:
Correct me if I am wrong but ME-EPI (BOLD fMRI) and ME-FLASH/GRE (anatomy) are two different sequences and for two different purposes. 
What you describe, @emdupre, is the former and hence I would add to `func` rather than `anat`. More importantly it is outside the scope of this BEP. I am not sure about the current state of the specs on ME fMRI, but a `_mebold` suffix might be considered similarly to our solution(s).

Reply:
Hey @emdupre, @tiborauer. I think these are two very different use cases. In the case of "anatomical" ME-GRE you really focus on a single image and underlying quantitative parameters, rather than a time series. Ergo, the quantitative T2* estimate itself is usually the measure-of-interest (e.g., in my own work we used it as a marker for iron concentrations in subcortex and the stria of Gennari in V1).

Reply:
@emdupre I thought that it was an anatomical scan that you were using to get weighted averaging from ME, sorry this is my bad. As @tiborauer noted, if it had to has a suffix in this spec, it would be something like `ME-EPI`. 

If this is only a sub-portion of your dataset, kept separately from the whole data solely for `weighted averaging` , maybe it still can be defined here with `MEEPI` suffix. Even if this is the case, it is more of a managerial decision, so I give @Gilles86 the floor :)

Reply:
I'm a manager now?! I hate those people! ;)

I would like to strive for consensus and propose the following:

Given the current spec of of BIDS 1.0, I would keep 'functional' and 'anatomical' data separate. 

So for multi-echo bold files, which are BOLD-weighted and use EPI-readouts, we keep using `func/sub-01_task-fingertap_echo-1_bold.nii.gz`. 
 
For anatomical files, which are usually very different in terms of sequence parameters (no EPI, higher resolution) and intended use (segmentation, quantification of tissue paramerters), we can then use `anat/sub-01_echo-1_MEGRE.nii.gz`, etc..

> If this is only a sub-portion of your dataset, kept separately from the whole data solely for weighted averaging , maybe it still can be defined here with MEEPI suffix. Even if this is the case, it is more of a managerial decision, so I give @Gilles86 the floor :)

As I understand it, the T2* decay is fitted on the very same set of time series that is later reweighted and denoised, correct, @emdupre? In that case, the suffix should clearly be `_bold`. Otherwise, I actually _still_ think it makes a lot of sense to keep these 'calibration' bold time series in the func-directory, because of their contrast and intended use. Maybe you could use an `acq-calibration` or `task-calibration`-key to indicate intended use case.

@tiborauer, @agahkarakuzu, @emdupre, you agree?

Reply:
+1

Reply:
Hey everyone @tiborauer @agahkarakuzu @Gilles86 @emdupre! Just a little 💯 from me!

* I like the `MEGRE` suffix for anatomical scans
* I like keeping the `_bold` suffix for the multi-echo functional scans

Sorry for the radio silence and thank you all for having these conversations 💖

Reply:
Reviewed: approve ✅

Reply:
That's fine by me ! Thanks for clarifying -- I just wanted to make sure the suggestion _wasn't_ to have something like `_mebold` for the functional runs. So I was just trying to make the analogy over to the functional files :) Sorry for being unclear, and thanks again !